ci/cloudbuild/builds/checkers.sh

```diff
@@ -203,7 +203,7 @@ time {
   done
 
   mapfile -t libraries < <(features::libraries)
-  for library in "${libraries[@]}" opentelemetry; do
+  for library in "${libraries[@]}" opentelemetry oauth2; do
```

Comment:
why not add `oauth2` to `ci/etc/full_feature_list` and thus `features::libraries()` ?

Because it would have to be hardcoded in the generator, and that would be kinda weird?

https://github.com/googleapis/google-cloud-cpp/blob/631c00ec32419fb88e201952c98fd92323f42840/generator/standalone_main.cc#L179

Reply:
Because I think we should remove `ci/etc/full_feature_list` in favor of `cmake/print-all-libraries.cmake` (which does not exist yet)